packages/clangml/clangml.4.0.1/opam

```diff
@@ -0,0 +1,33 @@
+opam-version: "2.0"
+maintainer: ["Thierry Martinez <thierry.martinez@inria.fr>"]
+authors: ["Thierry Martinez <thierry.martinez@inria.fr>"]
+bug-reports: "https://gitlab.inria.fr/memcad/clangml"
+homepage: "https://gitlab.inria.fr/memcad/clangml"
+doc: "https://gitlab.inria.fr/memcad/clangml"
+license: "BSD"
+dev-repo: "git+https://gitlab.inria.fr/memcad/clangml"
+synopsis: "OCaml bindings for Clang API"
+description: """
+clangml provides bindings to call the Clang API from OCaml.
+"""
+depends: [
+  "conf-libclang"
+  "conf-ncurses"
+  "conf-zlib"
+  "dune" {>= "1.10.0"}
+  "ppxlib" {>= "0.9.0"}
+  "stdcompat" {>= "10"}
+  "override" {>= "0.1.0"}
+  "ocaml" {>= "4.04.0"}
+  "ocamlfind" {build & >= "1.8.0"}
+  "ocamlcodoc" {with-test}
+  "pattern" {with-test}
+]
+build: [
+  ["./configure" "--prefix=%{prefix}%" "--with-llvm-config=%{conf-libclang:config}%"]
+  ["dune" "build" "-p" name "-j" jobs]]
+run-test: [["dune" "build" "@runtest"]]
```

Comment:
```suggestion
run-test: [["dune" "runtest" "-p" name "-j" jobs]]
```

Reply:
`dune build` alone without `-p` or `--profile=release` promotes some warnings as errors. This should be avoided in releases.

Reply:
Thanks! Should be fixed now.